src/ode_solver/relaxation_runge_kutta/algebraic_rrk_ode_solver.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __ENERGY_RRK_ODESOLVER_H__
+#define __ENERGY_RRK_ODESOLVER_H__
+
+#include "dg/dg_base.hpp"
+#include "rrk_ode_solver_base.h"
+#include "ode_solver/ode_solver_base.h"
+#include "ode_solver/runge_kutta_ode_solver.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// Relaxation Runge-Kutta ODE solver, calculating the relaxation parameter as in Ketcheson 2019
```

Comment:
the full reference would be helpful instead of author year

Reply:
added the full reference.